tests/lib/rules/prefer-user-event.test.ts

```diff
@@ -28,6 +28,26 @@ function createScenarioWithImport<
 
 const ruleTester = createRuleTester();
 
+function formatUserEventMethodsMessage(fireEventMethod: string): string {
```

Comment:
I reimplemented here the same method so I don't use the very same implementation. Otherwise, it would be tricky to catch errors formatting the message.